{{cookiecutter.project_slug}}/webpack/prod.config.js

```diff
@@ -0,0 +1,24 @@
+const { merge } = require('webpack-merge');
+const commonConfig = require('./common.config');
+
+// This variable should mirror the one from config/settings/production.py
+{%- if cookiecutter.use_whitenoise == 'n' and cookiecutter.cloud_provider == 'AWS' %}
+const s3BucketName = process.env.DJANGO_AWS_STORAGE_BUCKET_NAME;
+const awsS3Domain = process.env.DJANGO_AWS_S3_CUSTOM_DOMAIN ?
+    process.env.DJANGO_AWS_S3_CUSTOM_DOMAIN
+    : `${s3BucketName}.s3.amazonaws.com`;
+const staticUrl = `https://${awsS3Domain}/static/`;
+{%- elif cookiecutter.use_whitenoise == 'n' and cookiecutter.cloud_provider == 'GCP' %}
+const staticUrl = `https://storage.googleapis.com/${process.env.DJANGO_GCP_STORAGE_BUCKET_NAME}/static/`;
```

Comment:
This doesn't work currently with Docker, because the environment variables aren't set when this is config file is read (at build time).

Reply:
Has this been fixed?

Reply:
@charleshan 

If you are looking for a modern frontend solution based on Webpack for Django, I recommend you to check https://github.com/AccordBox/python-webpack-boilerplate

Thx